### ISSUE TITLE
0.29 release

### ISSUE THREAD

**Issue by agriyakhetarpal** [pyodide/pyodide-build#31]
It would be great if we could tag a release soon, with these PRs:

- #27
- #30
- #28 
- #29 

Would you like to include #21, too, or leave that for another release, @ryanking13?

and we could resolve this issue, too, perhaps? It looks quite easy to do and would help the conda-forge folks serve better metadata.

- #11 

I would probably leave out #14 for later, because it is slightly low priority for me to do (useful for some packages, but not a lot of packages)

I shall be available to help do the release – if it is needed of me to do so.

**Comment by ryanking13** [comment]
Sure, let's release 0.29 after resolving your open PRs. #21 would take some time, so it doesn't need to be included.

**Comment by ryanking13** [comment]
> I shall be available to help do the release – if it is needed of me to do so.

Releasing pyodide-build is quite easy (it is a benefit of unvendoring things :) ); you just need to update the changelog, and create a tag. So feel free to make a release when it is ready.

**Comment by agriyakhetarpal** [comment]
Awesome! I'll do it this week when I am ready. Thanks for the help in getting all those PRs merged!

**Comment by agriyakhetarpal** [comment]
Also, would it be okay to switch to using GitHub Releases from tags, instead of just tags? This improves our security practices, because one needs permissions in the repository to create a GitHub Release instead of just being able to push a tag (side note: we should add some [tag protection rules](https://github.blog/changelog/2022-03-09-tag-protection-rules/), if we haven't already). So, it would have the benefit of not triggering a PyPI release when a tag is pushed, so that we can use our trusted publishing better and make use of the GHA Attestations feature.

**Comment by ryanking13** [comment]
> Also, would it be okay to switch to using GitHub Releases from tags, instead of just tags?

Sure, fine with me.

**Comment by agriyakhetarpal** [comment]
Tagged: https://github.com/pyodide/pyodide-build/releases/tag/v0.29.0
GHA release workflow run: https://github.com/pyodide/pyodide-build/actions/runs/10941566243

However, the upload failed – this is because I moved the publishing job(s) to another workflow file (they are now in a dedicated `release.yml` instead of `main.yml`). I don't have access to the project on PyPI, so could you re-add the trusted publishing configuration in the PyPI settings, @ryanking13? I've deleted the release for now, and will update the tag once it is done.

**Comment by ryanking13** [comment]
@agriyakhetarpal I updated workflow file name in PyPI Trusted Publisher Management. Could you try rereleasing the package?

**Comment by agriyakhetarpal** [comment]
Thanks for the speedy reviews! We can close this now.
